docs/guides/streamlit_guide.rst

```diff
@@ -0,0 +1,53 @@
+
+After the threshold is selected, the user can make its own annotation, whether or not she or he thinks the predictions are correct. This is where the *human-in-the-middle* comes into play, by responding to a model made prediction with a user made annotation, that could eventually be used to provide feedback to the model or to make retrainings.
+
+Once the annotated labels are selected, the user can press the **log** button. A ``TextClassificationRecord`` will be created an logged into Rubrix with all the information about the process: the input text, the prediction and the annotation. This data is also displayed in the streamlit app, as the process ends. But you could always change the input text, the threshold or the annotated labels and log again!
+
+How to run the app
+------------------
+
+We've created a `standalone repository <https://github.com/recognai/rubrix-streamlit-example>`_\  for this streamlit app, for you to clone and play around. To run the app, follow these steps:
+
+#. Install the requirements into a fresh environment (or into your system, but take care with the dependency problems!) with Python 3, via ``pip install -r requirements.txt``.
+#. Run ``streamlit run app.py``.
+#. In the response prompt, streamlit will give you the localhost direction where your app will be running. You can now open it in your browser.
+
+Rubrix integration
+------------------
+
+Rubrix can be used alongside any third-party apps, as long as there is a Python script running beneath. In our case, the logging of the record is made when the log button is pressed. In that moment, two lists will be populated:
+
+
+* ``labels``\ , with the predicted labels by the zero-shot classifier
+* ``selected_labels``\ , with the annotated labels, selected by the user.
+
+Then, the ``TextClassificationRecord`` will be logged as follows:
+
+.. code-block:: python
+
+   item = rb.TextClassificationRecord(
+                   inputs={"text": text_input},
+                   prediction=labels,
+                   prediction_agent="typeform/squeezebert-mnli",
+                   annotation=selected_labels,
+                   annotation_agent="streamlit-user",
+                   multi_label=True,
+                   event_timestamp=datetime.datetime.now(),
+                   metadata={"model": "typeform/squeezebert-mnli"}
+               )
+
+   dataset_name = "multilabel_text_classification"
+
+   rb.log(name=dataset_name, records=item)
```

Comment:
```suggestion
   rubrix.log(name=dataset_name, records=item)
```

Reply:
We want rubrix.something to be 'the default', or we want to embrace the rb (somehow like pandas embrace the pd)?

Reply:
Ok, but then i think we should add an `import rubrix as rb` in the code block, otherwise it is not clear what `rb` is.

Reply:
okay, i could include that in all blocks and not only at the beggining